src/helpers/image-helper.php

```diff
@@ -205,7 +219,7 @@ public function get_attachment_variations( $attachment_id, $image_params = [] )
 	 *
 	 * @return string|null
 	 */
-	protected function get_first_usable_content_image_for_post( $post_id ) {
+	public function get_first_usable_content_image_for_post( $post_id ) {
```

Comment:
I will add a new method named `get_post_content_image` which wraps this method. I personally don't like the naming of this, because it just returns an image and who cares about it being the first usable one. You just want an image from the post content.